website/content/vagrant-cloud/boxes/architecture.mdx

```diff
@@ -0,0 +1,116 @@
+---
+layout: vagrant-cloud
+page_title: Vagrant Box Architecture
+description: "Vagrant box architecture and default architecture."
+---
+
+# Architecture for Vagrant boxes
+
+Providers for Vagrant boxes, in Vagrant version 2.4.0 and newer, can include 
+multiple architecture options. This allows you to have multiple instances 
+of one specific provider. The specific provider shares a common name for
+artifacts of different architectures.
+
+Using the `hashicorp/precise32` and `hashicorp/precise64` boxes as an example,
```

Comment:
It might be good to have a section title that indicates this is configuration the user writes. 

```suggestion
For example, the `hashicorp/precise32` and `hashicorp/precise64` boxes each
include a provider for `virtualbox`.
```